packages/streams/simple.js

```diff
@@ -187,8 +187,12 @@ const pipeStartByType = {
 
 function nmea2000input (subOptions, logging) {
   if (subOptions.type === 'ngt-1-canboatjs') {
+    const actisenseSerial = require('./actisense-serial')
```

Comment:
Doesn't the `require` call throw when it fails because of a missing `serialport`?

Reply:
No. There's a try/catch so that the rest of canboatjs can be used when serialport is missing.

Reply:
ok. Lint will barf on the next line though?

Reply:
the was the cause of the `(intermediate value) is not a function` error message reported the other day